packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -18,6 +19,8 @@ import {
 import { Button } from '@wordpress/components';
 import { plus } from '@wordpress/icons';
 
+const blockNavigationContext = { useBlockNavigationSlots: false };
```

Comment:
I suppose one final comment is that `use...` is often used for hooks these days. It probably wouldn't get confusing given it's clearly a property, but something like `hasBlockNavigationSlots` might be the safest option.